tools/profiles/debug.json

```diff
@@ -42,7 +42,7 @@
             "--diag_suppress=Pa050,Pa084,Pa093,Pa082", "-On", "-r", "-DMBED_DEBUG",
             "-DMBED_TRAP_ERRORS_ENABLED=1"],
         "asm": [],
-        "c": ["--vla"],
```

Comment:
My bad, I was getting compilation issues with exporters when enabling "--c++", "--no_rtti", "--no_exceptions" flags in *.json files. Compile flags and exporter flags didn't seemed synchronized. 

 "--no_rtti", "--no_exceptions"  - Not working with export tool. Enabling those flags directly in project settings work, but updating the template files didn't help as well. Verified now, the *.ewp file version for IAR8 is 3 and we are using version 2. Migration required.

Will remove this commit and add bug to mbed-os